src/dashboard/src/templates/archival_storage/archival_storage.html

```diff
@@ -55,7 +55,12 @@ <h1>
         </small>
       </h1>
 
-    <button id="create-aic-btn" class="btn btn-primary">{% trans "Create an AIC" %}</button>
+    {% blocktrans with "Create an AIC from current search results" as create_aic_title %}
+    <button
+      id="create-aic-btn"
+      class="btn btn-primary"
+      title="{{ create_aic_title }}">Create an AIC</button>
```

Comment:
That string is being translated already because it's within a [{% blocktrans %} block](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#blocktrans-template-tag), which is handy and was new to me!

Reply:
Based on some internal feedback, I've modified this so that we're not passing the whole function into the message files.